New version: Pluto v0.19.37

JuliaRegistrator opened the pull request:
- Registering package: Pluto
- Repository: https://github.com/fonsp/Pluto.jl
- Created by: @fonsp
- Version: v0.19.37
- Commit: 1e78c85a1d8e6fd5ebf596e9f704aa0f6928dcb6
- Reviewed by: @fonsp
- Reference: https://github.com/fonsp/Pluto.jl/commit/1e78c85a1d8e6fd5ebf596e9f704aa0f6928dcb6#commitcomment-137641836
- Description: 🎈 Simple reactive notebooks for Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd68fc830287cd088d6b7e170f3d3bd8839419f3957e82dcac5cb5e07ffe18ecf462ce50d5dfef356caf9789f22b1ae74b3c7285ec21b608cad0a5072bfa67670a8843d1d25b5f8df9ca456d1e6471ab58d3e10016e59e8522b00fcee847967295ccc3bf0487d57c1048493b5df8d8b6c1751292fa74aeee4e59d8e19349673325621a60a55eb3b756e80d4438b7565431a474b5f2bcc94d7256fbd0ce0c821530b22f08f6cb895525ddcbd8fb5a2f06441 -->

fonsp commented:
asfd

fonsp commented:
<img width="461" alt="image" src="https://github.com/JuliaRegistries/General/assets/6933510/9a925232-6a75-47e7-9ab9-f384bc389602">

JuliaRegistrator committed:
New version: Pluto v0.19.37

UUID: c3e4b0f8-55cb-11ea-2926-15256bba5781
Repo: https://github.com/fonsp/Pluto.jl.git
Tree: e1f12b2d765ff3f50e3732017126bbb5335b7911

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
